api/models/Events.js

```diff
@@ -0,0 +1,30 @@
+"use strict";
+
+var LinkingModels = require('sails-linking-models');
+var ModelHelpers = require('../lib/ModelHelpers');
+
+const model = {
+  schema: true,
+  autoCreatedAt: true,
+  autoUpdatedAt: true,
+  autoPK: false,
+  attributes: {
+    operation: {
+      type: 'string',
+      required: true,
+      enum: ['check in']
+    },
+    event_received: {
+      type: 'datetime',
+      required: true
+    },
+    person_id: {
+      type: 'string',
+      required: true,
+      primaryKey: true
```

Comment:
"can this not just point to the detainee model?" is how I began, but assumed the events model would want the actual person_id then subsequent Detainee queries would programmatically lookup the event record based on the person_id. I.e, the person_id would the binding element between the two, but it would be a manual process.